Field_Simulation/Simulation/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Threading;
+
+namespace WindowsFormsApp1
+{
+    public partial class Form1 : Form
+    {
+        Mutex cleanM = new Mutex();
+        List<int[]> toClean = new List<int[]>();
+        int vertical = 0;
+        const int CIRCLE_TIME = 1 * 60 ;
+        const int FIELD_SIZE = 50;
+        Bitmap bitmap;
+        DateTime time;
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            time = DateTime.Now;
```

Comment:
`this.time` and `time` are either alright, but you cannot use different practices in the same file. Look at like 31 for example, where you used the opposite practice.